eitprocessing/mixins/slicing.py

```diff
@@ -11,33 +11,22 @@
 
 class SelectByIndex(ABC):
     time: NDArray
```

Comment:
`SelectByIndex` does not require a time attribute. This should be moved to
`SelectByTime` instead.